src/strategies_and_params/replicastrategy.jl

```diff
@@ -49,8 +49,9 @@ NoStats(N=1) = NoStats{N}()
 
 replica_stats(::NoStats, _) = (), ()
 
+# TODO: implement guiding vector importance sampling
 """
-    AllOverlaps(n=2, operator=nothing) <: ReplicaStrategy{n}
+    AllOverlaps(n=2, operator=nothing, transform=nothing, vecnorm=true) <: ReplicaStrategy{n}
```

Comment:
```suggestion
    AllOverlaps(; n=2, operator=nothing, transform=nothing, vecnorm=true) <: ReplicaStrategy{n}
```
Please switch to keyword arguments. See below.